projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -654,23 +669,68 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     }
 
     /** @hidden @internal */
-    public handleClosing(event: CancelableEventArgs): void {
+    public handleClosing(event: CancelableBrowserEventArgs & IBaseEventArgs): void {
         this.onClosing.emit(event);
-    }
 
-    /** @hidden @internal */
-    public handleClosed(): void {
         if (this.value && this.value.start && !this.value.end) {
-            this.updateValue({ start: this.value.start, end: this.value.start });
+            this.value = { start: this.value.start, end: this.value.start };
         }
         if (this.value && !this.value.start && !this.value.end) {
-            this.updateValue(null);
+            this.value = null;
+        }
+
+        let outsideClick = true;
+        if (event.event) {
+            const elementRect: DOMRect = this.element.nativeElement.getBoundingClientRect();
+            if (elementRect.left < event.event.x && event.event.x < elementRect.right
+                && elementRect.top < event.event.y && event.event.y < elementRect.bottom) {
+                outsideClick = false;
+            }
+        }
+        if (outsideClick) {
+            // outside click
+            this.updateValidityOnBlur();
+        } else {
+            // input click
+            this.focusInput();
+        }
+    }
+
+    private updateValidityOnBlur() {
+        this.onTouchCallback();
+        if (this.hasProjectedInputs) {
+            this.projectedInputs.forEach(i => {
+                const ngControl = this._ngControl || (i as IgxDateRangeInputsBaseComponent).ngControl;
```

Comment:
Yeah, nope. Fairly certain this applies *only if* the range itself has `ngControl`, and that's still odd use for me